content/Day-11/_index.md

```diff
@@ -0,0 +1,40 @@
+
+![alt_text](images/image1.png "image_tooltip")
+ 
+### Understanding the underlying systems Splunk is monitoring
+
+**What is Unstructured Data?** This is data that is not in rows or columns. It is hard to read and manipulate the data and even harder to make sense of what might be in the dataset.  The following image shows an example of unstructured data from a Weblogic server. 
+
+![alt_text](images/image2.png "image_tooltip")
+
+From the image above, it is really difficult to determine if there is a problem. We would need to look closely at all the output to determine a reported problem. Now, imagine we have one-hundred (100) servers and that we manage all with their own unique problem/error. This becomes a problem no one’s what to deal with on a daily basis. This is one of the problems Splunk solves by bringing structure to unstructured data. 
+
+**What is Structured Data**? Structured data is the opposite of unstructured data and represents a clearer picture for us to read, work with and analyze data. It can be presented in rows and columns (tables or tabular data) and normally comes from a web server or event logs. Taking a look at the screenshot, you’ll notice a field **status : 200. **This is a common field that is an indicator that a webpage was returned to the user successfully. Other status codes like 400 series can be searched and presented to show us which pages did not return successfully. We can easily read a lot of the metadata about the request and response of the page using Splunk. 
+
+**Note**: Http response codes tell us  if a particular request failed or was successful. This topic is outside of the scope of this book but should be reviewed at some point in your career journey.   
+
+![alt_text](images/image3.png "image_tooltip")
+
+### Splunk Documentation & Community 
+
+**Splunk Documentation** has two main platforms under the Splunk umbrella; Splunk Enterprise and Splunk Cloud. Based on the product we have installed the documentation will be different. Splunk Cloud provides insights for data at the petabyte-scale data analytics across a hybrid cloud solution. Splunk Enterprise allows us to search, analyze and visualize data for actionable insights from all of your data within the enterprise. 
+
+### Splunk Community 
+
+Start here
```

Comment:
Is this intended to link to the splunk community page?

Reply:
added context for this section